middleware/test/unit/ContextTests.ts

```diff
@@ -186,11 +185,7 @@ describe('Context Tests', () => {
         const fsStat = new fs.Stats();
         sinon.stub(fsStat, 'isDirectory').returns(true);
         sinon.stub(fsStat, 'isFile').returns(true);
-        sandbox.stub(fs, 'statSync')
-          .withArgs(`/platform/services/${sname}/secret`)
-          .returns(fsStat)
-          .withArgs(`/platform/services/${sname}/secret/${key}`)
-          .returns(fsStat);
+        sandbox.stub(fs, 'statSync').returns(fsStat);
```

Comment:
Why the change here? Before the change any unexpected paths would make a test fail I think.